winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
+{% highlight xaml %}
+
+<Grid>
+<syncfusion:SfAvatarView />
+</Grid>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+// Creating an instance of the AvatarView control.
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+Initials content type is used to show initials to the `AvatarView`. When using the `ContentType` as initials, you need to set the initial character using the following properties:
+
+* `InitialsType` - Defines the type of characters to be displayed.  
+* `AvatarName` - Defines the initials to be displayed.
+
+### InitialsType
+
+The `InitialsType` contains the following two types:
+
+* SingleCharacter
+* DoubleCharacter
+
+You must set the `AvatarName` string property for displaying the initials value in the AvatarView.
+
+**SingleCharacter**
+
+The `SingleCharacter` is used for displaying the first character in the string you have set in the `AvatarName` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+             
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_initials_avatarview.png)
+
+**DoubleCharacter**
+
+The `DoubleCharacter` is used for displaying a two-characters text you have set in the `AvatarName` property. If the initials consist of a single word, it will show the first and last letters of that word. If the initials contain two or more words, it will display the first letter of the first word and the first letter of the second word.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_doubleinitials_avatarview.png)
+
+## Custom Image
+
+You can add a custom image by setting the `ImageSource` property.
+
```

Comment:
@MadhumithanJayapathi You can add any custom image as the avatar by using the `ImageSource` property.

Reply:
@Manivannan-E Changed the content of the custom image as mentioned.